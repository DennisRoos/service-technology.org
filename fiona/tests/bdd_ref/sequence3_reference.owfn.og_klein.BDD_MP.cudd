.ver DDDMP-2.0
.mode A
.varinfo 4
.dd bddMp
.nnodes 28
.nvars 8
.nsuppvars 8
.ids 0 1 2 3 4 5 6 7
.permids 6 2 1 0 5 7 3 4
.nroots 1
.rootids -28
.nodes
1 1 0 0
2 7 1 -1
3 6 1 -2
4 5 1 3
5 4 1 4
6 6 1 2
7 5 1 6
8 4 7 1
9 3 5 8
10 6 2 -1
11 5 10 -1
12 4 1 -11
13 6 2 1
14 5 3 13
15 4 4 14
16 3 12 15
17 2 9 16
18 1 1 17
19 5 3 1
20 4 19 1
21 3 20 5
22 5 13 3
23 5 1 -10
24 4 22 23
25 3 24 8
26 2 21 25
27 1 26 1
28 0 18 27
.end
