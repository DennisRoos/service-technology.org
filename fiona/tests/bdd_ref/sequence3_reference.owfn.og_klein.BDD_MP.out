digraph "DD" {
size = "7.5,10"
center = true;
edge [dir = none];
{ node [shape = plaintext];
  edge [style = invis];
  "CONST NODES" [style = invis];
" 0 " -> " 1 " -> " 2 " -> " 3 " -> " 4 " -> " 5 " -> " 6 " -> " 7 " -> "CONST NODES"; 
}
{ rank = same; node [shape = box]; edge [style = invis];
"F0"; }
{ rank = same; " 0 ";
"cf7";
}
{ rank = same; " 1 ";
"cf5";
"cd9";
}
{ rank = same; " 2 ";
"cd8";
"cc0";
"cf4";
}
{ rank = same; " 3 ";
"d4c";
"cd7";
"d1a";
"d30";
"cf3";
"d3d";
}
{ rank = same; " 4 ";
"cf2";
"cd6";
"d3c";
"d19";
"d4b";
"d2f";
}
{ rank = same; " 5 ";
"d2e";
"cea";
"d45";
"ce2";
"cd0";
"ce7";
"d18";
"d09";
}
{ rank = same; " 6 ";
"cf6";
"ce1";
"d27";
"d08";
}
{ rank = same; " 7 ";
"ce0";
}
{ rank = same; "CONST NODES";
{ node [shape = box]; "cb8";
}
}
"F0" -> "cf7" [style = dotted];
"cf7" -> "cd9";
"cf7" -> "cf5" [style = dashed];
"cf5" -> "cf4";
"cf5" -> "cc0" [style = dashed];
"cd9" -> "cb8";
"cd9" -> "cd8" [style = dashed];
"cd8" -> "cd7";
"cd8" -> "d3d" [style = dashed];
"cc0" -> "d30";
"cc0" -> "d4c" [style = dashed];
"cf4" -> "cf3";
"cf4" -> "d1a" [style = dashed];
"d4c" -> "d4b";
"d4c" -> "cb8" [style = dashed];
"cd7" -> "cb8";
"cd7" -> "cd6" [style = dashed];
"d1a" -> "d19";
"d1a" -> "cb8" [style = dashed];
"d30" -> "cb8";
"d30" -> "d2f" [style = dashed];
"cf3" -> "cb8";
"cf3" -> "cf2" [style = dotted];
"d3d" -> "d3c";
"d3d" -> "cb8" [style = dashed];
"cf2" -> "cea";
"cf2" -> "ce7" [style = dotted];
"cd6" -> "cd0";
"cd6" -> "d09" [style = dashed];
"d3c" -> "d09";
"d3c" -> "ce2" [style = dashed];
"d19" -> "cb8";
"d19" -> "d18" [style = dashed];
"d4b" -> "ce7";
"d4b" -> "d45" [style = dashed];
"d2f" -> "cb8";
"d2f" -> "d2e" [style = dotted];
"d2e" -> "d27";
"d2e" -> "cf6" [style = dotted];
"cea" -> "d27";
"cea" -> "cb8" [style = dotted];
"d45" -> "cf6";
"d45" -> "cb8" [style = dashed];
"ce2" -> "cb8";
"ce2" -> "ce1" [style = dashed];
"cd0" -> "cb8";
"cd0" -> "d27" [style = dotted];
"ce7" -> "ce1";
"ce7" -> "cb8" [style = dashed];
"d18" -> "d08";
"d18" -> "cf6" [style = dashed];
"d09" -> "d08";
"d09" -> "cb8" [style = dashed];
"cf6" -> "cb8";
"cf6" -> "ce0" [style = dotted];
"ce1" -> "cb8";
"ce1" -> "ce0" [style = dashed];
"d27" -> "ce0";
"d27" -> "cb8" [style = dotted];
"d08" -> "ce0";
"d08" -> "cb8" [style = dashed];
"ce0" -> "cb8";
"ce0" -> "cb8" [style = dotted];
"cb8" [label = "1"];
}
